src/components/NewCommentForm.tsx

```diff
@@ -75,25 +210,40 @@
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classnames('textarea', {
+              'is-danger': formState.errors.body,
```

Comment:
Great use of the `classnames` library to conditionally apply classes based on the form state.